README.md

```diff
@@ -1,2 +1,167 @@
-# Full-Stack-Capstone-Backend
-Explore the backend of Doctor Reservation App! This repository, powered by Rails, manages user authentication, reservation logic, and API endpoints.
+# Full Stack Capstone Backend<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍